### DISCUSSION TITLE
4.1.6 maintenance release

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#2152] [General]
All DEVs,

I plan to roll a last maintenance release for the 4.1.x series. After I cherry pick important, small and safe fixing commits from master, I'll post the proposed release notes here. But if you would like to see a specific fix into 4.1.6, please let me know.

**Comment by mobab-th** [comment]
Some changes that need to be translated?

**Reply by lfcnassif** [reply]
Not for now, I usually try to keep changes with pending translations out of minor/fix releases. If needed, I'll let you know. Thank you @mobab-th for trying to help!

**Reply by lfcnassif** [reply]
Hi @mobab-th. If we don't include anything else, the only one new expression to translate is "Affected evidence".

**Comment by lfcnassif** [comment]
Finished copying dozens of fixes from master to 4.1.6, I'll still run tests. Here are the proposed changes:
https://github.com/sepinf-inc/IPED/blob/4.1.x/ReleaseNotes.txt

Should we include something else?

PS: I included a simple but significant memory optimization when processing UFDR files, and also a RAR expansion speed optimization because it was mixed with a partial RAR expansion fix.

**Reply by lfcnassif** [reply]
I'm runnig regression tests. @marcus6n I would like your help to test the 4.1.6 release candidate, which snapshot was created here: https://github.com/sepinf-inc/IPED/actions/runs/8547041407

Please process your test evidences with it and with 4.1.5 version, enabling all modules. Please compare each module results, they should be the same.

**Reply by marcus6n** [reply]
Right, I'll test 4.1.6 release candidate.

**Reply by marcus6n** [reply]
@lfcnassif I tested using [snapshot 4.1.6](https://github.com/sepinf-inc/IPED/actions/runs/8547041407) and 4.1.5, and I noticed that the results of the modules remained the same in both versions.

**Reply by lfcnassif** [reply]
All tests seem good, just published the 4.1.6 release.
